airbyte-integrations/connectors/source-mongodb/secrets.sample/incremental_configured_catalog.json

```diff
@@ -0,0 +1,69 @@
+{
+  "streams":[
+    {
+      "sync_mode": "full_refresh",
```

Comment:
for the tests to correctly verify incremental functionality this should be set to `incremental` with the correct cursor field

Reply:
Changed one stream to incremental